pkg/client/src/app/pages/applications/analysis-wizard/components/add-custom-rules.tsx

```diff
@@ -119,8 +119,9 @@ export const AddCustomRules: React.FunctionComponent<IAddCustomRules> = ({
           if (validateXMLFile(data as string))
             handleReadSuccess(data as string, file);
           else {
-            // TODO propagate xml validation error
-            const error = new DOMException("Not a valid XML");
+            const error = new DOMException(
+              `File ${file.name} is not a valid XML`
```

Comment:
Small tweak - can we pull out the XML error message from the validation library itself here? Seems like that would be useful info to display.

Reply:
Yeah that's a good idea as it seems to be a limitation of initial requirements.  
I saw you used it in your patch and that's on my to do list